tests/test-definitions.txt

```diff
@@ -90,7 +90,7 @@ resilience_failover priority=750 cluster -- --dumpAgencyOnError true
 resilience_move priority=600 cluster -- --dumpAgencyOnError true
 resilience_sharddist cluster -- --dumpAgencyOnError true
 
-recovery_cluster priority=2000 cluster buckets=8 -- --test search
```

Comment:
Why this change? Is it established that the non-search recovery tests work in cluster?

Reply:
the cluster suite now does that filtering: 
```
                                       // At the moment only view-tests supported by cluster recovery tests:
                                      ).filter(testname => testname.search('search') >= 0);
```
since I don't think that we should abuse test filtering for this.